### PR TITLE
chore!: Official v1 release

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

This should set the next version of the CLI to `v1.0.0`

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "cli": "1.3.0-pre.1",
+  "cli": "0.9.9",
```
Comment (erezrokah) [inline .release-please-manifest.json:2]: This tells release please the current version, so with a breaking change the next one should be `v1.0.0`

File: cli/Makefile [hunk 2]
```diff
@@ -4,4 +4,4 @@ lint:
 
 .PHONY: test
 test:
-	go test ./...
\ No newline at end of file
+	go test ./...
```
Comment (erezrokah) [inline cli/Makefile:7]: This is a stub change to trigger a release in the monorepo (as we detect changes based on paths in the repo)

File: .release-please-manifest.json (hunk 1 above)
Comment (hermanschaaf) [reply .release-please-manifest.json:2]: Don't our breaking changes usually get their minor version bumped? Or how does it work now?

**Review by yevgenypats — APPROVED** [review]

File: .release-please-manifest.json (hunk 1 above)
Comment (erezrokah) [reply .release-please-manifest.json:2]: Yes we did than until https://github.com/cloudquery/cloudquery/pull/2050/files

Comment (hermanschaaf) [reply .release-please-manifest.json:2]: Ah cool, thanks 👍

**Pull request merged**